db/primary_migrate/20230113202352_create_sp_return_logs_returned_at_date_issuer_index.rb

```diff
@@ -0,0 +1,20 @@
+class CreateSpReturnLogsReturnedAtDateIssuerIndex < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def up
+    safety_assured do
+      execute <<-SQL
+        CREATE INDEX CONCURRENTLY index_sp_return_logs_on_returned_at_date_issuer
+        ON public.sp_return_logs
+        USING btree ((returned_at::date), issuer)
+        WHERE (billable = true);
```

Comment:
This is not strictly needed but might make for a smaller index, re: https://github.com/18F/identity-idp/pull/7604#pullrequestreview-1241275053

Reply:
I think it's worth it, yeah